source/content/start-states/wordpress.md

```diff
@@ -14,6 +14,16 @@ For the most part, [Pantheon's WordPress upstream](https://github.com/pantheon-s
 
 ## Latest Release
 
+### 2022-08-16
```

Comment:
I imagine we'll hold this update for the next WP core release, so this date will likely change. That's a product decision for @scottbuscemi 😄

Reply:
The [previous update](https://pantheon.io/docs/start-states/wordpress#2022-07-12) didn't necessarily align with a WordPress release (or, if it did, it wasn't noted), so I was following that precedent.

Reply:
In fact, none of the other updates in the release notes mention a WordPress version besides 6.0 (when themes broke), but I guess the change doesn't actually get applied until a WordPress release goes out.

Reply:
just confirming that this PR is still good to go and that this convo here is "in the future"

🚀 👍 ?

Reply:
We actually chatted about this on our retro today. We need to have another conversation about it when @scottbuscemi is around, so let's hold off for now (unless there's a WordPress release that sneaks out that the change would go to users with).